### DISCUSSION TITLE
JS SDK - Login with Refresh or Access Token ?

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#9810] [Feature Requests]
After receiving an directus_access_token which can be exchanged for an refresh_token (and access_token) I want to use the js sdk. But this package does not allow me to login with the tokens. There is a function directus.auth.static(token) but as I tried it, that was not possible to use.
Even tried to implement my own Storage, but this does not to affect the token.

After: https://github.com/directus/directus/discussions/9347 and https://github.com/directus/directus/discussions/8867

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
